### ISSUE TITLE
[Screen Reader - AML Data Drift - Create Monitors] Label is not associated with “Threshold” slider on “Monitor Setting” page.

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#11933]
*User Experience: *
User who depends upon screen reader won't get information related to label if label is not associated with “Threshold” slider. Screen reader will not provide any information about the label which will create difficulties to users.

*Note:* User credentials should NOT be included in the bug.
*Repro Steps:*
1. Open Url: https://master.ml.azure.com/data/monitor?wsid=/subscriptions/35f16a99-532a-4a47-9e93-00305f6c40f2/resourcegroups/data4ml-demo-rg/workspaces/data4ml-demo&tid=72f988bf-86f1-41af-91ab-2d7cd011db47 and login via a valid credential
2. Dataset Monitors screen will open.
3. Navigate to Create Monitor button and select it.
4. Enter all valid Basic Info Screen and Select next button.
5. Navigate to “Threshold” slider appearing under Monitor Setting Screen.
6. Verify the issue.
 
*Actual Result:*
Label is not associated with “Threshold” slider on “Monitor Setting” page. Screen reader does not provide any information about the label.

 
*Expected result:*
Label should be associated with “Threshold” slider on “Monitor Setting” page so that screen reader user can find out the information about the label.

**Note:**
This issue also exist under Chrome + JAWS & Chrome + NVDA.

**MAS Reference:**
MAS 1.3.1 - Info and Relationships (40)

**Reference Links**

    Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
    External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
    Please reach out to C&AI Teams channel for any process related queries.

![MAS1 3 1_Label is not associated with “Threshold” slider on “Monitor Setting” page](https://user-images.githubusercontent.com/30456910/74321114-c64c7c80-4da7-11ea-941d-64d8bc2844ed.png)
